### ISSUE TITLE
Questions in filter_relabel module in the code

### ISSUE THREAD

**Issue by hanjiang-mufeng** [rtmaww/SENT#12]
Thank you very much for your excellent work. I have some questions in your code. I would appreciate it if you could answer them.

        train_batch_data = self.train_dl.dataset
        for i in range(len(train_batch_data)):
            train_batch_data[i][-1][:] = self.ori_train_labels[i]
        self.train_dl = DataLoader(train_batch_data, batch_size=self.batch_size, sampler=SequentialSampler(train_batch_data))

Why do you use the original label to re label the data in every time noise filtering and relabeling? Won't this operation make the iteration meaningless?

**Comment by zh190920** [comment]
> Thank you very much for your excellent work. I have some questions in your code. I would appreciate it if you could answer them.
> 
> ```
>     train_batch_data = self.train_dl.dataset
>     for i in range(len(train_batch_data)):
>         train_batch_data[i][-1][:] = self.ori_train_labels[i]
>     self.train_dl = DataLoader(train_batch_data, batch_size=self.batch_size, sampler=SequentialSampler(train_batch_data))
> ```
> 
> Why do you use the original label to re label the data in every time noise filtering and relabeling? Won't this operation make the iteration meaningless?



> Thank you very much for your excellent work. I have some questions in your code. I would appreciate it if you could answer them.
> 
> ```
>     train_batch_data = self.train_dl.dataset
>     for i in range(len(train_batch_data)):
>         train_batch_data[i][-1][:] = self.ori_train_labels[i]
>     self.train_dl = DataLoader(train_batch_data, batch_size=self.batch_size, sampler=SequentialSampler(train_batch_data))
> ```
> 
> Why do you use the original label to re label the data in every time noise filtering and relabeling? Won't this operation make the iteration meaningless?

hi, Do you know why it works like this., why are the original labels reassigned each iteration.thanks for you reply
